assets/css/sensei-course-theme/focus-mode.scss

```diff
@@ -1,92 +1,113 @@
+	0% { transform: translateX(0) scale(1) rotate(180deg); }
+	50% { transform: translateX(-3px) scale(1.05) rotate(180deg); }
+	100% { transform: translateX(0) scale(1) rotate(180deg); }
 }
-@media screen and (min-width: 782px) {
-	.sensei-course-theme__sidebar {
-		padding-top: 32px;
-	}
+
+@keyframes sensei-learning-mode-expand {
+	0% { transform: translateX(0) scale(1); }
+	50% { transform: translateX(3px) scale(1.05); }
+	100% { transform: translateX(0) scale(1); }
 }
 
-.sensei-course-theme--focus-mode {
-	--focus-header-height: 45px;
+@media screen and (min-width: $breakpoint ) {
+	.sensei-course-theme__focus-mode-toggle {
+		.sensei-course-theme--focus-mode & {
+			position: absolute;
+			left: 20px;
+		}
+		padding: 4px;
+		white-space: nowrap;
+		display: flex;
+		flex-direction: row-reverse;
+		align-items: center;
+		gap: 12px;
+		border: none;
+		background-color: var(--bg-color);
 
-	.sensei-course-theme {
-		&__header {
-			&__container {
-				height: 39px;
-				overflow: hidden;
-			}
-			.sensei-course-theme-course-progress, .sensei-course-theme-prev-next-lesson-text {
-				display: none;
-			}
-			.wp-block-site-logo {
-				margin-right: -12px;
-				img {
-					max-height: 0;
-				}
-			}
-			.wp-block-sensei-lms-course-title {
-				font-size: 16px;
-			}
-			.sensei-course-theme-course-progress-bar {
-				height: 6px;
-			}
-			.sensei-course-theme__button {
-				font-size: 12px;
-				padding: 3px 8px;
-				font-weight: 700;
-			}
+		&__disable {
+			display: none;
 		}
 
-		&__sidebar {
-			left: -400px;
-			overflow: visible;
+		.sensei-course-theme__focus-mode-toggle-icon {
+			width: 10px;
+			height: 10px;
+			transform: rotate(180deg);
 		}
-		&__main-content {
-			margin-left: 0 !important;
+
+		&:hover {
+			.sensei-course-theme__focus-mode-toggle-icon {
+				animation: sensei-learning-mode-collapse 0.8s infinite;
+			}
 		}
-		&__focus-mode-toggle {
-			left: 420px;
-			flex-direction: row;
-			&__enable {
-				display: none;
+	}
+
+	.sensei-course-theme--focus-mode {
+		--header-height: 45px;
+
+		.sensei-course-theme {
+			&__header {
+				&__container {
+					height: 39px;
+					overflow: hidden;
+				}
+				.sensei-course-theme-course-progress, .sensei-course-theme-prev-next-lesson-text {
+					display: none;
+				}
+				.wp-block-site-logo {
+					margin-right: -12px;
+					img {
+						max-height: 0;
+					}
+				}
+				.wp-block-sensei-lms-course-title {
+					font-size: 16px;
+				}
+				.sensei-course-theme-course-progress-bar {
+					height: 6px;
+				}
+				.sensei-course-theme__button {
+					font-size: 12px;
+					padding: 3px 8px;
+					font-weight: 700;
+				}
 			}
-			&__disable {
-				display: revert;
+
+			&__sidebar {
+				left: -400px;
+				overflow: visible;
 			}
-			.sensei-course-theme__focus-mode-toggle-icon {
-				transform: none;
+			&__main-content {
+				margin-left: 0 !important;
+			}
+			&__focus-mode-toggle {
+				top: 0;
+				left: 400px;
+				width: 100vw;
```

Comment:
Instead of the full-width expand button with a background, what if we ditch the label, and just show the icon
<img width="17" alt="image" src="https://user-images.githubusercontent.com/176949/151460910-4c200cc4-a0e2-4050-ae59-8e45c6fcc108.png"> in the top right corner when collapsed? There is padding enough around the content that it won't overlap, and the user already interacted it with it so won't need an extra explanation. (Just add 'Expand' as a `title` attribute)

Reply:
With this approach, the button seems a little weird! But I liked your other idea about a button in the header. See the other comment https://github.com/Automattic/sensei/pull/4684#discussion_r794092400.

<img width="782" alt="Screen Shot 2022-01-28 at 10 31 07" src="https://user-images.githubusercontent.com/876340/151560740-e45fcc7c-9614-48e6-9de8-64125da5f22e.png">